addons/hr/static/src/fields/boolean_radio.js

```diff
@@ -27,25 +35,29 @@ export class BooleanRadio extends RadioField {
     }
 
     get value() {
-        if (this.type === "boolean") return this.props.record.data[this.props.name].toString();
+        if (this.type === "boolean")
+            return this.props.record.data[this.props.name].toString();
```

Comment:
nitpick, clearer with standard brackets
```suggestion
        if (this.type === "boolean") {
            return this.props.record.data[this.props.name].toString();
        }
```